src/frontend/src/utils/validateDerivationOrigin.ts

```diff
@@ -1,12 +1,8 @@
+import { resolveCanisterId as resolveCanisterIdFn } from "$src/utils/canisterIdResolution";
```

Comment:
Did you do this to be able to test the function?

Reply:
Yes. I could also provide a wrapper function so that caller don't need to provide the `canisterIdResolution` as a parameter. Should I do that / change something here?

Reply:
Maybe you could add the imported function as default parameter and substitute it in the tests.

This way you don't need to change the calls anywhere.